_posts/2023-02-26-pernStack.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 1st Full Stack Webapp with PERN Stack
+layout: post
+banner_image: https://images.unsplash.com/photo-1522202176988-66273c2fd55f?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1471&q=80
+---
+
+The team first full stack application journey
+
+My FE team is tasked with creating a simple full stack application. We decided to use PERN stack because our knowledge of Javascript which would covered 3 out of 4 tech in the stack, with combination of these tech allowed us to build a full stack web application with CRUD operations.
+
+## Why we choose PERN Stack
+
+PERN is an acronym for PostgreSQL, Express, React and Node.
+[![](https://mermaid.ink/img/pako:eNpNTU1LxDAQ_SthTgrd0iRN0w2yh0WvIuhJchnadD9okpKkuGvpfzeriDKHeW_exyzQ-d6AgmH0H90RQyJve-2CwS6V50geNpsdebpMwcSYuXZ_-E7DP0E78pybbhDuf2IvPqZD1rW7DRRgTbB46vO3JduJhnQ01mhQGfZmwHlMGrRbsxXn5F-vrgOVwmwKmKcek3k84SGgBTXgGPN1Qvfuvf01ZQpqgQsoyUveUCmYYFII3vICrqBqVm4lk4wyUefNmVgL-PwuqMqGVU3D2i2tWdVWlK5fdjZYXA?type=png)](https://mermaid.live/edit#pako:eNpNTU1LxDAQ_SthTgrd0iRN0w2yh0WvIuhJchnadD9okpKkuGvpfzeriDKHeW_exyzQ-d6AgmH0H90RQyJve-2CwS6V50geNpsdebpMwcSYuXZ_-E7DP0E78pybbhDuf2IvPqZD1rW7DRRgTbB46vO3JduJhnQ01mhQGfZmwHlMGrRbsxXn5F-vrgOVwmwKmKcek3k84SGgBTXgGPN1Qvfuvf01ZQpqgQsoyUveUCmYYFII3vICrqBqVm4lk4wyUefNmVgL-PwuqMqGVU3D2i2tWdVWlK5fdjZYXA)
+
+PostgreSQL - It is an Object-Relational Database management system that uses the SQL language combined with many other features that safely store and scale data.
```

Comment:
"object-relational database" not "Object-Relational Database"


I think I'd also leave out the "many other features that safely store and scale data" piece.  It feels too salesy.  It seems like the primary goal is to talk about why you thought Postgres served *your needs*, not the general needs provided by their website.  The answer here can totally be "because it was easy to pick up" or "because we had some prior exposure to it".